src/runners/alert_queries_runner.py

```diff
@@ -157,12 +176,35 @@ def query_for_alerts(query_name: str):
     process_results(results, ctx, query_name)
 
 
+def record_metadata(ctx, metadata):
+    metadata['RUN_START_TIME'] = str(metadata['RUN_START_TIME'])   # We wantd them to be objects for mathing
+    metadata['RUN_END_TIME'] = str(metadata['RUN_END_TIME'])       # then convert to string for json serializing
+    metadata['RUN_DURATION'] = str(metadata['RUN_DURATION'])
+
+    statement = f'''
+        INSERT INTO {METADATA_TABLE}
+            (event_time, v) select '{metadata['RUN_START_TIME']}',
+            PARSE_JSON(column1) from values('{json.dumps(metadata)}')
+        '''
+    try:
+        log.info("Recording run metadata...")
```

Comment:
same as before — either period ending or "done." after it's done?

Reply:
fixed!